pkg/redislocker/redislocker.go

```diff
@@ -0,0 +1,148 @@
+package redislocker
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"golang.org/x/exp/slog"
+
+	"github.com/tus/tusd/v2/pkg/handler"
+)
+
+var (
+	LockExchangeChannel = "tusd_lock_release_request_%s"
```

Comment:
It would be helpful the users could configure a prefix for these keys, so they scope them and avoid collision with other services.